google-http-client/src/main/java/com/google/api/client/http/HttpRequest.java

```diff
@@ -876,7 +876,7 @@ public HttpResponse execute() throws IOException {
       executeException = null;
 
       // run the interceptor
-      if (executeInterceptor != null) {
+      if (executeInterceptor == null) { // This code is wrong. Just for testing.
         executeInterceptor.intercept(this);
```

Comment:
google-cloud-storage's test threw `NullPointerException` correctly.

<img width="1341" alt="Screen Shot 2021-05-05 at 17 00 57" src="https://user-images.githubusercontent.com/28604/117208763-7f3f1280-adc3-11eb-8485-fe12efe3c6a5.png">

Reply:
However this approach does not work for Gradle projects such as "LTS Library Test / lts-library-test (beam)".